doc/nrf/device_guides/nrf70/nrf7002eb_dev_guide.rst

```diff
@@ -10,20 +10,92 @@ Developing with nRF7002 EB
 The nRF7002 :term:`Expansion Board (EB)` (PCA63561), part of the `nRF70 Series Family <nRF70 Series product page_>`_, can be used to provide Wi-Fi® connectivity to compatible development or evaluation boards through the nRF7002 Wi-Fi 6 companion IC.
 For example, you can use it with the :ref:`Nordic Thingy:53 <ug_thingy53>`, an IoT prototyping platform from Nordic Semiconductor.
 
+You can also use the nRF7002 :term:`Expansion Board (EB)` (PCA63561) to provide Wi-Fi® connectivity to the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15`.
+
 .. figure:: images/nRF7002eb.png
    :alt: nRF7002 EB
 
    nRF7002 EB
 
+PCB edge connector and castellated edge holes
+*********************************************
+
 The nRF7002 EB features a :term:`Printed Circuit Board (PCB)` edge connector and castellated holes to provide Wi-Fi connectivity through the nRF7002 companion IC.
 
+The castellated holes on the side of the board allow the EB to be used as a breakout board that can be soldered to other PCB assemblies.
+This way, you can provide Wi-Fi capabilities to develop Wi-Fi applications with another System on Chip (SoC), MPU, or MCU host.
+For the pinout of the castellated holes, see `nRF7002 EB castellated edge holes`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
+
 In case of the Nordic Thingy:53, the PCB edge connector connects nRF7002 to nRF5340, which acts as a host.
 For the pinout of the PCB edge connector, see `nRF7002 EB PCB edge connector`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
 For information about how to connect to the Nordic Thingy:53, see `Using nRF7002 EB with the Nordic Thingy:53`_, also in the hardware guide.
 
-The castellated holes on the side of the board allow the EB to be used as a breakout board that can be soldered to other PCB assemblies.
-This way, you can provide Wi-Fi capabilities to develop Wi-Fi applications with another System on Chip (SoC), MPU, or MCU host.
-For the pinout of the castellated holes, see `nRF7002 EB castellated edge holes`_ in the board's `Hardware User Guide <nRF7002 EB User Guide_>`_.
+
+For nRF54H20 DK and for nRF54L15 PDK following are the pin mapping:
+
```

Comment:
```suggestion

Pin mapping for the nRF54H20 DK and the nRF54L15 PDK
****************************************************

Given that the hardware documentation for the the nRF54H20 DK and the nRF54L15 PDK is not yet publicly available, refer to the following tables for the pin mapping for these kits:
```

Reply:
Is it important for the user to know that the hardware docs are not available yet?